corehq/apps/app_manager/templates/app_manager/partials/modules/case_list.html

```diff
@@ -111,7 +111,7 @@ <h4 class="panel-title panel-title-nolink">
       {% if request|toggle_enabled:'CASE_LIST_TILE' %}
         {% with app|get_available_modules_for_case_tile_configuration:module as available_modules %}
           {% if available_modules %}
-            <div class="row">
+            <div class="row" data-bind="visible: persistTileOnForms">
```

Comment:
Should this be `persistTileOnForms` or `persistTileOnForms()` or both work?
Seeing other usages of this in the code and I see both implementations.
https://github.com/dimagi/commcare-hq/blob/1b7f6454413694bf167d4f1ead7d5458c93b70aa/corehq/apps/app_manager/templates/app_manager/partials/modules/case_list.html#L136

Reply:
Both work. I'm surprised that the code you linked to works - but I just retested and it does.

My understanding is that if you provide an observable, knockout will use that, and if you provide a statement, it'll wrap that in an computed observable, so that code would become `ko.computed( return persistTileOnForms() || persistentCaseTileFromModule; )` which would always return true since `persistentCaseTileFromModule` will always exist. But perhaps it's smart enough to do something like `computed( return persistTileOnForms() || persistentCaseTileFromModule(); )`?

I'm fairly certain I've had problems before with not including the parens and having the statement always return true. But I'm confident that when you're just using a single observable, `myObservable` and `myObservable()` are equivalent.

Reply:
Thanks @orangejenny 
Relying on KO to figure things out, does sound dangerous. I'd vote to use `()` always for observable though since that is how we get the value for the observable in JS.
Though, not something to cater to in this PR.